Meadowlark-js/tests/system/crud/SystemTestSetup.ts

```diff
@@ -89,3 +93,9 @@ export function academicWeekBodyClient1(): FrontendRequest {
     }`,
   };
 }
+
+export function configureEnvironmentForMongoSystemTests(): void {
+  process.env.DOCUMENT_STORE_PLUGIN = MONGO_DOCUMENT_STORE_PLUGIN;
+  process.env.SIGNING_KEY = TEST_SIGNING_KEY;
+  process.env.MEADOWLARK_DATABASE_NAME = 'meadowlark_system_tests';
+}
```

Comment:
One thought I had was for DOCUMENT_STORE_PLUGIN to be set external to the tests. That way they have no dependency on the particular document store, so the system tests for the various document stores could be run as npm scripts that just differ in the enviroment variables, e.g. `cross-env DOCUMENT_STORE_PLUGIN=@edfi/meadowlark-mongodb-backend yarn jest system --runInBand`

Reply:
We'll definitely want to refactor in that direction.